boards/common/esp_common/Makefile.dep

```diff
@@ -0,0 +1,7 @@
+# enable bootloader reset over USB, requires CDC ACM to be used
+ifneq (,$(filter stdio_cdc_acm stdio_tinyusb_cdc_acm,$(USEMODULE)))
+  USEMODULE += usb_board_reset
+  USEMODULE += esp32_bootloader_reset
```

Comment:
This dependency could be removed if it wouldn't be a separate module. See my comment below.
```suggestion
```